<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="plot_1" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1251</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>1251</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot_3" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>1251</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>600</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>580</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X Coord</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>620</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Coord</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>670</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>580</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>t min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>620</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>t max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>590</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Energy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btplot">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>650</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btsearch">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>660</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="compute">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>660</y>
      <width>471</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>570</y>
      <width>20</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>570</y>
      <width>20</width>
      <height>151</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="time_search">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>600</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xcoord">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>580</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ycoord">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>620</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="baterylvl">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>660</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_tmin">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>580</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_tmax">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>620</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="integral">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>620</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1289</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
